### ISSUE TITLE
PT_PTF_Setup_Guide Inaccurate Link & Problematic Instructions for PTFeeder/GnomeFeeder

### ISSUE THREAD

**Issue by blockheadbits** [CryptoGnome/Gnome-Feeder#18]
Step 6 on the guide "PT_PTF_Setup_Guide (3).docx" does not contain an accurate link to the PTFeeder and only directs users to the CryptoGnome Profit Trailer repo.  PTFeeder instructions need some updating and clarification.

After browsing through the repo, I figured that I'm supposed to go to Gnome-Feeder to get the PTFeeder settings; however, it is very unclear how the version of GnomeFeeder compares to the official PTFeeder release.  Should users essentially treat the latest release of official PTFeeder [as an update](https://github.com/mehtadone/PTFeeder/wiki/Updating-PT-Feeder) whereby they would copy over all newest files to GnomeFeeder and leave only config and database files in GnomeFeeder?  Or should GnomeFeeder always be treated as a stand-alone item apart from PTFeeder?

**Comment by CryptoGnome** [comment]
I keep the entire folder up to date with the latest PT Release for ease of use and updating if you gitpull.

Also I use different settings in some other files that are needed!

**Comment by blockheadbits** [comment]
Thanks for quick response!

Also I saw you are working on ETH settings, which is great.  I tried
modifying the low-balance settings from BTC to ETH by changing min volume,
max cost, and one or two other things to mirror ETH and it quickly bought a
bunch of pairs that then went pretty quickly into DCA mode.

Haven't had time to trouble shoot, so I'm not sure where I went wrong.  It
could just be because of the crash today/last night; however, I'd think
that PTFeeder should prevent buying during a crash by going into
superbear.  Fwiw, I did not attempt to mod any of the settings for
superbear, bear, boring, bull, or superbull.
